src/components/shared/Link.js

```diff
@@ -0,0 +1,13 @@
+import { Anchor as GrommetAnchor, Box } from 'grommet'
+
+export const Link = ({ underline = false, ...props }) => {
```

Comment:
Id like to have this component automatically open to a new page if the `href` starts with `http`

Reply:
The [Anchor](https://github.com/AlexsLemonade/refinebio-web/blob/c06a2081f4b498784436e625a09cb07726dee81f/src/components/shared/Anchor.js) component (which utilizes both Next.js `Link` and Grommet `Anchor` and performs internal/external urls check) was created in the PR [29 - Homepage (Without Cached Data)](https://github.com/AlexsLemonade/refinebio-web/pull/32).  Thus, I've added the `target="_blank"` attribute to that component instead of this one. 

After the PR [29 - Homepage (Without Cached Data)](https://github.com/AlexsLemonade/refinebio-web/pull/32) is merged to `dev`, I'll pull this change to all the newer PRs accordingly, thank you!